Source/Core/BoxGeometry.js

```diff
@@ -1,28 +1,29 @@
 /*global define*/
 define([
-        './BoundingSphere',
```

Comment:
Make sure WebStorm is inserting spaces for tabs.  We did not need to change the formatting here.  We just wanted to add a reference for `deprecationWarning`.